src/components/helpers/validation.js

```diff
@@ -1,6 +1,25 @@
 import moment from 'moment';
 import { defineMessages } from 'react-intl';
+import isNumeric from 'validator/lib/isNumeric';
 
+export const isPositiveInteger = n =>
+  typeof n !== 'undefined' &&
+  (typeof n === 'number' || isNumeric(n)) &&
+  parseInt(n) > 0;
+
+export const isNegativeInteger = n =>
+  typeof n !== 'undefined' &&
+  (typeof n === 'number' || isNumeric(n)) &&
+  parseInt(n) > 0;
```

Comment:
```suggestion
  parseInt(n) < 0;
```
Negative integer is smaller than 0